### ISSUE TITLE
GetMoviesAsunc issue

### ISSUE THREAD

**Issue by Afy65** [LordMike/TMDbLib#270]
Hi
I get the error below a fair bit form using the following:

```
 TMDbClient client = new TMDbClient("c6b31d1cdad6a56a23f0c913e2482a31");
            var config = await client.GetConfigAsync();
            //get the full movie details
            var movieData = await client.GetMovieAsync(movieID, TMDbLib.Objects.Movies.MovieMethods.ExternalIds | TMDbLib.Objects.Movies.MovieMethods.Videos | TMDbLib.Objects.Movies.MovieMethods.Credits);
```
Newtonsoft.Json.JsonReaderException: 'Unexpected character encountered while parsing value: <. Path '', line 0, position 0.'

Perhaps you could shed some light in this for me

Many Thanks

Daniel

**Comment by LordMike** [comment]
Sounds like a HTML response to me. Maybe TMDb had a failure on their end.

**Comment by Afy65** [comment]
Maybe they have some problems because it is doing all over the application that I am writing now? Even where that part of the app is finished and has worked great before.

**Comment by LordMike** [comment]
Yea. It does look like it would be html - so their gateway or front end is spewing out errors instead of api responses.

I would make a ticket with tmdb asking about it and asking if they could deliver a Json error instead of html.

Mike.

________________________________

**Comment by FrederikBolding** [comment]
Could possibly be fixed with #194

**Comment by LordMike** [comment]
Fixed in #275 - we should longer try to process HTML responses.
